src/ciel.lisp

```diff
@@ -397,7 +397,9 @@ We currently only try this with serapeum. See *deps/serapeum/sequences-hashtable
 (defparameter *current-pprint-indentation* 1
   "We use custom indentation instead of the pretty printer, because it doesn't print correctly in the shell (indentations are way too large).")
 
-;TEST
+(pushnew :ciel *features*)
```

Comment:
I add this one, which isn't related to alex issue, for writing ciel script. So emacs will know when my slime inside the `ciel-user` package. 

feel free to change it if you have different ideas.

Reply:
Hey I have a couple `(push :ciel ciel-user::*features*)` in scripting.lisp, they should do the job? 

What was your use case when inside Emacs?

related: we also have `termp:termp` to detect if we are in a "DUMB" terminal (Emacs) or in a real one, to quit or not.

Reply:
I think it is the render issue. My use case is start the slime, and `(ql:quickload "ciel")`, then switch to package `ciel-user`. And in this these [code](https://github.com/ccqpein/AdventOfCode/blob/1380c97338e6b5960beb66a543e2a130852e290c/tools/download-input.lisp#L94-L95), emacs show it as grey. After I add `(pushnew :ciel *features*)` in `ciel.lisp`. emacs show it as feature enable code.

Reply:
So, the `ciel` feature is used in the script to determine if the environment is in script mode or not. For normal development processes, like when I compile a script file, `#+ciel` shouldn't be enabled.

It might have been a misunderstanding to add `:ciel` to the `ciel-user` package.

Maybe we should change the feature name from `:ciel` to`:ciel-script` in `script.lisp`?